node/nodejs_intro/README.md

```diff
@@ -169,6 +231,21 @@ What do we think the output to screen will be now?
   <summary>
     Solution
   </summary>
-   The output will be identical to the previous example. The reason: Our setTimeout is moved into the WebAPI regardless of
-  how quickly it is set to resolve.
+
+
+
+    Hello,
+    Yoda
+    I am
+
+   The output will be identical to the previous example. The reason: Regardless, of how quickly our setTimeout is set to resolve (in this case, in 0 seconds), it is still moved into the WebAPI.
 </details>
+<p>&nbsp;</p>
+
+
```

Comment:
Due to the complexity of this topic and that it requires a fair amount of studying and practicing before things begin to click for learners, I would add something like a `key takeaways`  bullet list something that students should be able to grasp at this point:
- be able to better predict the order of functions based on the event loop 
- A single threaded language means one function runs at a time
- An asynchronous language means that the order in which functions are called are not necessarily the order they will resolve

Just keep it as simple as possible

Reply:
Sounds good!